Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM python:3.10-alpine
-RUN apk add --no-cache bash
```

Comment:
I have absolutely no idea why, but the alpine issue continued to have the Cython error issue even after upgrading PyYaml